tests/test_casbin_enforcer.py

```diff
@@ -146,6 +149,21 @@ def item():
     assert rv.status_code == status
 
 
+@pytest.mark.parametrize(
+    "header_string, expected_list",
+    [
+        ("noexist,testnoexist,users  ", ["noexist", "testnoexist", "users"]),
+        ("noexist testnoexist users", ["noexist", "testnoexist", "users"]),
+        ("noexist, testnoexist, users", ["noexist", "testnoexist", "users"]),
+        ("somegroup, group with space", ["somegroup", "group with space"]),
+        ("group with space", ["group with space"])
+    ]
+)
+def test_sanitize_group_headers(header_string, expected_list):
+    header_list = CasbinEnforcer.sanitize_group_headers(header_string)
+    assert header_list == expected_list
+
+
```

Comment:
Why is this part here? I believe we already have `test_sanitize_group_headers` at line 239

Reply:
I missed this.. thanks for pointing it out, it's deleted now and as this was a duplicated test, it was not running at all, so that is why the test you pointed out in your other comment was not failing.